molgenis-core-ui/package.json

```diff
@@ -33,10 +33,13 @@
     "jquery-validation": "1.14.0",
     "jquery.cookie": "^1.4.1",
     "jquery.fancytree": "2.14.0",
-    "moment": "2.9.0",
+    "moment": "2.11.2",
```

Comment:
This is a bit of an issue, until we merge this to master they'll require different versions of moment and that's awkward cause the pull request builder runs them in the same node_modules install.